dotnet/Getting_Started_VS/Getting-Started-VS/Program.cs

```diff
@@ -0,0 +1,168 @@
+﻿/*
+Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+SPDX-License-Identifier: Apache-2.0
+
+
+Purpose
+
+  The purpose of this example is to verify that you can connect to your AWS
+  account and perform simple read operations on your AWS objects. This example
+  reads the names of your RDS DB instances and your S3 buckets. If you don't
+  have any DB instances or buckets, the example still connects to your AWS
+  account and reports what it doesn't find.
+
+  This code example was created by using Visual Studio 2019
+  This code example uses .NET Core 3.1 to create a cross-platform application
```

Comment:
Or can line 15 & 16 be combined? 
"This code example uses .NET...cross-platform application that is a console application."

Reply:
Let's talk about this one later.